latex/properties.tex

```diff
@@ -190,7 +190,7 @@ \subsection{Ledger State Properties for Delegation Transitions}
   \begin{multline*}
     \forall \var{l}, \var{l'} \in \ledgerState: \applyFun{validLedgerstate}{l}\\
     \implies \forall \var{c} \in \DCert, \var{l} \trans{delegw}{c} \var{l'}
-    \wedge \RegKey{c} \implies \applyFun{author}{c} = \var{hk}\\ \implies
+    \wedge var{c}\in\DCertRegKey{c} \implies \applyFun{author}{c} = \var{hk}\\ \implies
```

Comment:
```suggestion
    \wedge \var{c}\in\DCertRegKey{c} \implies \applyFun{author}{c} = \var{hk}\\ \implies
```

Reply:
in all those cases we might also simply have `\var{c} \in \DCertRegKey{c}` in the beginning of the hypothesis, not fist element of `\DCert` and then specifying the subset.